faucet/valve.py

```diff
@@ -1036,10 +1036,15 @@ def lldp_handler(self, now, pkt_meta, other_valves):
                 self.logger.info('LLDP on %s, %s from %s (remote %s, port %u) state %s' % (chassis_id, port, pkt_meta.eth_src, valve_util.dpid_log(remote_dp_id), remote_port_id, remote_port_state))
             port.dyn_lldp_beacon_recv_state = remote_port_state
 
+        exp_remote_src = self.dp.ports[port.number].lldp_peer_mac
```

Comment:
exp_ is usually a bad prefix because it often means "experimental" -- maybe rename just peer_mac_src or something like that?  peer_src_mac?